experiments/AtmosLES/rising_bubble_bryan.jl

```diff
@@ -0,0 +1,253 @@
+using ClimateMachine
+ClimateMachine.init(parse_clargs = true)
+
+using ClimateMachine.Atmos
+using ClimateMachine.Orientations
+using ClimateMachine.ConfigTypes
+using ClimateMachine.Diagnostics
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.SystemSolvers
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.Thermodynamics
+using ClimateMachine.TemperatureProfiles
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.VariableTemplates
+using ClimateMachine.NumericalFluxes
+using ClimateMachine.VTK
+
+using StaticArrays
+using Test
+using Printf
+using MPI
+
+using CLIMAParameters
+using CLIMAParameters.Atmos.SubgridScale: C_smag
+using CLIMAParameters.Planet: R_d, cp_d, cv_d, MSLP, grav
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+# ------------------------ Description ------------------------- #
+# 1) Dry Rising Bubble (circular potential temperature perturbation)
+# 2) Boundaries - `All Walls` : Impenetrable(FreeSlip())
+#                               Laterally periodic
+# 3) Domain - 20000m[horizontal] x 10000m[vertical] (2-dimensional)
+# 4) Timeend - 1000s
+# 5) Mesh Aspect Ratio (Effective resolution) 2:1
+# 7) Overrides defaults for
+#               `init_on_cpu`
+#               `solver_type`
+#               `sources`
+#               `C_smag`
+# 8) Default settings can be found in `src/Driver/Configurations.jl`
+# ------------------------ Description ------------------------- #
+function init_risingbubble!(problem, bl, state, aux, (x, y, z), t)
+    FT = eltype(state)
+    R_gas::FT = R_d(bl.param_set)
+    c_p::FT = cp_d(bl.param_set)
+    c_v::FT = cv_d(bl.param_set)
+    γ::FT = c_p / c_v
+    p0::FT = MSLP(bl.param_set)
+    _grav::FT = grav(bl.param_set)
+
+    xc::FT = 10000
+    zc::FT = 2000
+    r = sqrt((x - xc)^2 + (z - zc)^2)
+    rc::FT = 2000
+    θ_ref::FT = 300
+    Δθ::FT = 0
+
+    if r <= rc
+        Δθ = FT(2) * cospi(0.5 * r / rc)^2
+    end
+
+    #Perturbed state:
+    θ = θ_ref + Δθ # potential temperature
+    π_exner = FT(1) - _grav / (c_p * θ) * z # exner pressure
+    ρ = p0 / (R_gas * θ) * (π_exner)^(c_v / R_gas) # density
+    q_tot = FT(0)
+    ts = LiquidIcePotTempSHumEquil(bl.param_set, θ, ρ, q_tot)
+    q_pt = PhasePartition(ts)
+
+    ρu = SVector(FT(0), FT(0), FT(0))
+
+    #State (prognostic) variable assignment
+    e_kin = FT(0)
+    e_pot = gravitational_potential(bl.orientation, aux)
+    ρe_tot = ρ * total_energy(e_kin, e_pot, ts)
+    state.ρ = ρ
+    state.ρu = ρu
+    state.ρe = ρe_tot
+    state.moisture.ρq_tot = ρ * q_pt.tot
+end
+
+function config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
+
+    # Choose explicit solver
+    solver_type = MultirateInfinitesimalStep
+    fast_solver_type = LowStorageRungeKutta2N
+
+    if solver_type == MultirateInfinitesimalStep
```

Comment:
@vogel-d So I would like to try to simplify this script before we merge this. Can we choose a single configuration for this example and just use that? For example, just use MIS with an ARK fast solver?

Reply:
Of course! I just used this for testing and to have an overview over the different fast solvers, while coding. 

I however noticed that after merging the master, rising_bubble_bryan.jl does not run anymore. 
In line 44, as far as I can tell, instead of 
`function init_risingbubble!(problem, bl, state, aux, (x, y, z), t)`
the coordinates have to be specified as
`function init_risingbubble!(problem, bl, state, aux, localgeo, t)`
`(x, y, z) = localgeo.coord`

Additionaly in line 69
`ts = LiquidIcePotTempSHumEquil(bl.param_set, θ, ρ, q_tot)`
LiquidIcePotTempSHumEquil does not seem to be defined anymore. But I don't know, with what it has to be replaced.

Reply:
OK great! I'm fixing this now